### DISCUSSION TITLE
nft marketplace

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#457] [Q&A] [answered]
Hello,

Could you give me example how to create simple nft marketplace without opensea?

Please help

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello there, open sea reads the uri info linked to the connected wallet, it means you can see the NFTs you own but they are not necessarily linked to opensea. If you want a deep on tutorial please check chainlink official YouTube channel there are a bunch of tutorials including nft ones: https://youtube.com/c/chainlink

Also, a market place is more frontend related so I suggest you to use react with a framework like usedapp.

Best regards, Cromewar.
